meson_config.h

```diff
@@ -109,6 +109,9 @@
 /* Define to 1 if you have the `backtrace_symbols' function. */
 #mesondefine HAVE_BACKTRACE_SYMBOLS
 
+/* Define to 1 if dbtob impllementation is broken. */
```

Comment:
Small typo in code comment: _implementation_ ;)

Reply:
Fixed!